rocAL/include/parameters/parameter_random.h

```diff
@@ -73,6 +93,13 @@ class UniformRand : public Parameter<T> {
         _end = end;
         return 0;
     }
+
+    void create_array(unsigned batch_size) override {
+        if (_array.size() == 0)
+            _array.resize(batch_size);
+        _batch_size = batch_size;
```

Comment:
Can we change the name to size instead ?
As we can use this for video pipeline as well and in that case it would be bs * seq_len.